pkg/config/testdata/config.toml

```diff
@@ -170,7 +170,7 @@ secrets = "env(AUTH_SEND_SMS_SECRETS)"
 # Configure one of the supported SMS providers: `twilio`, `twilio_verify`, `messagebird`, `textlocal`, `vonage`.
 [auth.sms.twilio]
 enabled = true
-account_sid = "account_sid"
+account_sid = "env(SECRET_TWILLIO_SID)"
```

Comment:
**remark**

This should not trigger an `env()` error from the `config.Load` anymore since this is declared in the `dynamic_env.value`